independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -447,6 +447,10 @@ protected Class<?> findClass(String moduleName, String name) {
         }
     }
 
+    protected URL findResource(String name) {
+        return getResource(name);
+    }
+
```

Comment:
Looks like we are also missing `Enumeration<URL> findResources(String name)` impl.